### DISCUSSION TITLE
Running demo without `gdown`

### DISCUSSION THREAD

**Discussion by Dref360** [ServiceNow/azimuth#46] [Show and tell]
It can be cumbersome for users to install `gdown` locally to download the demo.

As a workaround, you can do the following:

1. Download the data manually from [Google Drive](https://drive.google.com/file/d/1LmP_Y1KGxiOOJrTkXjocLnOgSmhqDRzk/view?usp=sharing).
2. Extract content at `azimuth_shr/files/clinc-demo`.
3. Run the demo as before `make CFG_PATH=/config/development/clinc_dummy/conf.json compose`

You should have the following directory tree:

```bash
azimuth_shr
├── data
├── files
│   └── clinc-demo
│       ├── CLINC150_trained_model
│       └── oos-eval
│           └── data
```

Hope that help!

**Comment by gabegma** [comment]
Alternatively, if the error you had while downloading `gdown` was the the one below, please refer to this [stackoverflow](https://stackoverflow.com/questions/55833509/attributeerror-type-object-callable-has-no-attribute-abc-registry) thread.

`AttributeError: type object 'Callable' has no attribute '_abc_registry'`
